jobsets/default.nix

```diff
@@ -1,34 +1,111 @@
-{ nixopsPrsJSON ? ./simple-pr-dummy.json
+############################################################################
+# This is the jobset declaration evaluated by Hydra to dynamically
+# generate jobsets.
+#
+# The arguments for this file come from spec.json.
+#
+# See also the Hydra manual:
+#   https://github.com/NixOS/hydra/blob/master/doc/manual/declarative-projects.xml
+#
+############################################################################
+
+{ nixpkgs ? <nixpkgs>
+, declInput ? {}
+
+# Paths to JSON files containing PR info fetched from github.
+# An example file is ./simple-pr-dummy.json.
+, nixopsPrsJSON ? ./simple-pr-dummy.json
 , cardanoPrsJSON ? ./simple-pr-dummy.json
 , daedalusPrsJSON ? ./simple-pr-dummy.json
 , plutusPrsJSON ? ./simple-pr-dummy.json
 , logClassifierPrsJSON ? ./simple-pr-dummy.json
 , ouroborosNetworkPrsJSON ? ./simple-pr-dummy.json
 , chainPrsJSON ? ./simple-pr-dummy.json
-, nixpkgs ? <nixpkgs>
-, declInput ? {}
-, handleCardanoPRs ? true
+, walletPrsJSON ? ./simple-pr-dummy.json
 }:
 
-# Followed by https://github.com/NixOS/hydra/pull/418/files
+let pkgs = import nixpkgs {}; in
```

Comment:
we may want to update `spec.json`, this is importing a nixpkgs from the 16.09 channel